test/packs/ARM/RteTestGenerator/0.1.0/ARM.RteTestGenerator.pdsc

```diff
@@ -187,6 +187,12 @@
     <component generator="RteTestExternalGenerator" Cclass="RteTestGenerator" Cgroup="Check Global Generator" Cversion="0.9.0">
       <description>Component addressing external</description>
     </component>
+    <component generator="RteTestGeneratorIdentifier" Cvendor="ARM" Cclass="Device" Cgroup="RteTest Generated Component" Csub="RteTestOverlap" Cversion="1.1.0">
+      <description>Component with overlaping file lists</description>
+      <files>
+        <file category="sourceC" name="Src/RteTestConfig.c" attr="config" version="1.1.0"/>
```

Comment:
attr="config" is not supported for component files in gpdsc: the files  are alsready in project workspace and should not be copied

Reply:
While that might be so, this is how I could easily make the crash visible.
If this part is removed, make sure that csolution still crashes on the same statement as without it to ensure that the reported crash is corrected.